src/views/Home.vue

```diff
@@ -234,7 +264,18 @@ export default defineComponent({
           .replace('NISec', 'NIoSec')
           .replace('PFSec', 'PlFSec');
 
-        const planData = newPath
+        const [ hash, query ] = newPath.split('?');
+
+        if (query != undefined) {
```

Comment:
If url query does not contain "startSemester", we just skip this code. But shouldn't we make sure to reset the selected semester? Or write the selected semester to the url.

Reply:
This is only a problem, when manually editing the url (eg removing startSemester) and then pressing enter. Refreshing or loading with an url not containing that query param does not pose a problem.